src/Volume.ts

```diff
@@ -27,7 +28,34 @@ export interface ImageInfo {
     translation: [number, number, number];
     rotation: [number, number, number];
   };
+  channelGui: ChannelGuiOptions[];
```

Comment:
I suspect this is a hack that is only here for the demo viewer and probably shouldn't pollute the ImageInfo.  I think the channelGui objects could be stored entirely in the "state" object in the demo viewer.  You could do a quick check in the website-3d-cell-viewer codebase for any use of `channelGui`.  I doubt you'll find it...